### PR TITLE
Allow user override of input format to random backend.

### PR DESCRIPTION
**Pull request by Tilps**

### CODE REVIEW

File: src/neural/network_random.cc [hunk 1]
```diff
@@ -108,7 +106,10 @@ class RandomNetwork : public Network {
   RandomNetwork(const OptionsDict& options)
       : delay_ms_(options.GetOrDefault<int>("delay", 0)),
         seed_(options.GetOrDefault<int>("seed", 0)),
-        uniform_mode_(options.GetOrDefault<bool>("uniform", false)) {}
+        uniform_mode_(options.GetOrDefault<bool>("uniform", false)),
+        capabilities_{static_cast<pblczero::NetworkFormat::InputFormat>(
+                          options.GetOrDefault<int>("input_mode", 1)),
```
Comment (borg323) [inline src/neural/network_random.cc:111]: I think this should be `pblczero::NetworkFormat::INPUT_CLASSICAL_112_PLANE` instead of 1?

Comment (Tilps) [reply src/neural/network_random.cc:111]: They are the same value, but I can make the line longer I guess, sure :P

Comment (Tilps) [reply src/neural/network_random.cc]: Done.

Comment (borg323) [reply src/neural/network_random.cc]: I wasn't 100% sure the first time I was reading this, so had to check the ,proto file.

**Pull request merged**